README.md

```diff
@@ -1,30 +1,137 @@
+# Prerequisites
+
+[Playwright](https://playwright.dev/) requires [Node.js](https://nodejs.org/en/download/package-manager).
 
 # Installing Playwright
 
+Clone the repository and execute the following commands:
+
 ```bash
+cd playwright-runbook
 npm install
+npx playwright install
 ```
 
-# Set .env
+# Configuring Environment Variables
+
+Rename `.env.sample` to `.env`, and update the environment values.
+
+For local testing, set the URL to `localhost` or `candig.docker.internal`. All the credentials can be found in the `CanDIGv2/tmp/keycloak` folder.
 
-Rename `.env.sample` to `.env` and modify the values
+# Running Tests
```

Comment:
Some more detail about running the tests would be helpful. Does this assume you have a running local CanDIG stack? Do you need to run the playbook from a specific directory? Can you run it against a remote stack?